frontend/public/locales/en/renew.json

```
@@ -141,9 +141,19 @@
       "delegate": "I am renewing on behalf of someone else as a <strong>legal delegate or Power of Attorney</strong>"
     },
     "error-message": {
-      "type-of-application-required": "Select who this renewal is for"
+      "type-of-renewal-required": "Select who this renewal is for"
     },
     "back-btn": "Back",
     "continue-btn": "Continue"
+  },
+  "renewal-delegate": {
+    "page-title": "Renewing on behalf of someone as a legal delegate or Power of Attorney",
+    "contact-representative": "You must speak to a Service Canada representative if you are renewing on behalf of someone as their legal delegate or Power of Attorney. You can contact Service Canada by calling 1-833-537-4342 or by visiting a <contactServiceCanada>Service Canada Centre</contactServiceCanada> near you.",
```

```suggestion
    "contact-representative": "You must speak to a Service Canada representative if you are renewing on behalf of someone as their legal delegate or Power of Attorney. You can contact Service Canada by calling <noWrap>1-833-537-4342</noWrap> or by visiting a <contactServiceCanada>Service Canada Centre</contactServiceCanada> near you.",
```

Same for "French"... I guess.